eng/pipelines/coreclr/exploratory.yml

```diff
@@ -2,8 +2,8 @@
 trigger: none
 
 schedules:
-- cron: "0 14 * * 0"
-  displayName: Sun at 6:00 AM (UTC-8:00)
+- cron: "0 14 * * 0,6"
+  displayName: Sat and Sun at 6:00 AM (UTC-8:00)
```

Comment:
Wouldn't Wednesday or Thursday be better choices than Saturday, to have the runs more evenly spread out throughout the week?

Reply:
No. I don't want this to hog CI resources when people are at work.

Reply:
With fewer commits going in on weekends, I am not sure how running it on Saturday and Sunday would help.

Reply:
It's not about finding newly introduced regressions, it's about finding existing bugs in .NET 8 features. I particularly want to spend more time fuzzing to catch bugs in physical promotion.